Make.inc

```diff
@@ -655,6 +655,11 @@ $(error Pre-SSE2 CPU targets not supported. To create a generic 32-bit x86 binar
 pass 'MARCH=pentium4'.)
 endif
 
+ifneq ($(findstring $(JULIA_CPU_TARGET),generic i386 i486 i586 i686 pentium pentium2 pentium3),)
```

Comment:
We don't want to reject `generic`; that's what we actually use on `aarch64`.

Reply:
Good catch. Ideally we should check whether the arch is x86 and reject `generic` only in that case, but given that it's only for 0.6 and we have a more reliable solution on master, let's go with the simpler fix.

Reply:
Thanks, but at this point is it really worth taking the risk of breaking things on 0.6? Maybe better go with the more limited patch.